### ISSUE TITLE
Can't rescue downed colonist in vanilla (Harmony & RimThreaded)

### ISSUE THREAD

**Issue by UndeadKitten** [cseelhoff/RimThreaded#398]
**Describe the bug**
Nothing happens when I try to rescue this downed colonist (the colonists ignore the order), it repeats the last 2 errors in the error log each time I click Rescue. Reloading the save didn't fix it, but restarting the game then loading the save did.
EDIT: Version was RT 1.2.3

**To Reproduce**
1. Colonist is downed due to Muscle parasites and Food poisoning (major)
2. Select a different colonist
3. Right click downed colonist
4. Select the Rescue option
5. See error

**Error Log**
Error log & player.log:
https://gist.github.com/UndeadKitten/f4a16c2a9edc46b590bd75aa21cb917c
Save file:
https://drive.google.com/file/d/15tidWsE_vWuYRWgO9HpGzPhffhTyPRST/view?usp=sharing

**Comment by bababooeyy** [comment]
Doesnt seem to happen to me(playing in 1.2.4)

Save File: 
https://drive.google.com/file/d/1bIhkdotlcFLQZQVKZSnc6SGkKgA3g0jl/view?usp=sharing

Video of doing it(so you can tell me if im doing it wrong):



https://user-images.githubusercontent.com/77920755/106148254-cc538d00-61b3-11eb-8d3b-c11e8d2fd5d2.mp4

**Comment by UndeadKitten** [comment]
I have continued playing on the same save and I don't think I've encountered this exact error that I know of. I disabled dev-mode for a portion of my playtime as the frequent log popups can get annoying. Since restarting my game at the time fixed this error I don't know of any way to reproduce it.

(Below is copy and pasted across a few of my posts)

I am not able to reliably reproduce any of the errors that I get as far as I know, they happen seemingly at random while playing. Sometimes almost immediately upon loading up the save and sometimes hours later. I get many different errors, more than what I have posted here. I regularly get inconsequential errors and sometimes ones that break functions of the game (usually pawn unable to reach area) that are fixed upon restarting the game. Both on 1.2.3 and 1.2.4 preview.

If you have any suggestions for me on what to do then let me know.
